### DISCUSSION TITLE
Is it possible to merge two projects?

### DISCUSSION THREAD

**Discussion by hexpwn** [NationalSecurityAgency/ghidra#5889] [Q&A]
At some point I lost some data in my project. I want to merge a backup file into the current project. Is this possible?

**Comment by astrelsky** [comment]
I don't think so. However, you can export programs individually using the lossless gzf format (I think). Then just import them into the other project. It might be a bit tedious but I don't think there is another way.

**Reply by hexpwn** [reply]
thank you. I think doing that is the best option. Import the program and then use the diffing tool.
